screens/MovieStack/MovieListScreen.tsx

```diff
@@ -2,30 +2,34 @@ import React, { useEffect, useState } from "react";
 import { TouchableOpacity, StyleSheet } from "react-native";
 import { ActivityIndicator, FlatList } from "react-native";
 import { Text, View } from "react-native";
-import fetchAPI from "../fetchApi";
-import { Movie } from "../types";
+import fetchAPI from "../../fetchApi";
+import { Movie } from "../../types";
 import Accordion from "react-native-collapsible/Accordion";
 import { Ionicons } from "@expo/vector-icons";
 
-//---------------------------------------------------------
+//#region -------------------------------------------------
 export default function MovieListScreen({ navigation }) {
     const [movies, setMovies] = useState<Array<Movie>>([]);
     const [state, setState] = useState<Array<number>>([]);
+    const [errorMessage, setErrorMessage] = useState<string>(undefined);
 
     useEffect(() => {
         async function loadMovies() {
-            const json = await fetchAPI.getMovies();
-            setMovies(json);
+            setErrorMessage(undefined);
+            try {
+                const json = await fetchAPI.getMovies();
+                setMovies(json);
+            } catch (err) {
+                setErrorMessage("Sorry, we can't fetch your API");
+            }
         }
         loadMovies();
     }, []);
 
     return (
         <View style={styles.container}>
-            <TouchableOpacity onPress={() => navigation.openDrawer()}>
-                <Ionicons name="md-menu" size={32} color="black" />
-            </TouchableOpacity>
-            <Text style={styles.title}>Movies</Text>
+            {errorMessage ? <Text> {errorMessage}</Text> : <Text></Text>}
```

Comment:
Lepiej chyba w takim przypadku nie wyświetlać nic dla pustego bledu
```suggestion
            {!!errorMessage ? <Text> {errorMessage}</Text>}
```

**(bardzo ważna uwaga: napisanie `{errorMessage ? ...}` jest błędem - jesteś w stanie zgadnąć dlaczego? :) )**